package.json

```diff
@@ -21,6 +21,8 @@
     "clean": "polkadot-dev-clean-build",
     "demo:identicon:react": "yarn build:ts && webpack-serve --config packages/react-identicon/webpack.config.js --content packages/react-identicon --port 8080",
     "demo:identicon:vue": "yarn build:ts && webpack-serve --config packages/vue-identicon/webpack.config.js --content packages/vue-identicon --port 8080",
+    "example:rn:ios": "yarn build:split:code && cd packages/exampleReactNative && yarn copy-workspace-packages && cd ios && pod install && cd .. && yarn ios && yarn start",
+    "example:rn:android": "yarn build:split:code && cd packages/exampleReactNative && yarn copy-workspace-packages && yarn android && yarn start",
```

Comment:
We probably need to add an `example:rn:packager` or something simmilar - 

- build everything via `yarn build:split:code` (probably doesn't need to be in ios & android)
- cd into directory
- run `yarn start`
-

Reply:
Additionally we don't need the `yarn start` on either the `example:rn:android` or `example:rn:ios` commands. (Just assume that the user, as per docs, runs the `example:rn:packager` in a separate terminal)